packages/express-openapi/index.ts

```diff
@@ -116,8 +116,6 @@ export function initialize(args: ExpressOpenAPIArgs): OpenAPIFramework {
       let middleware = [].concat(ctx.additionalFeatures);
 
       if (operationDoc && ctx.allowsFeatures) {
-        middleware.unshift(createAssignApiDocMiddleware(apiDoc, operationDoc));
```

Comment:
I moved this way up so that `consumesMiddleware` has access to `req.operationDoc`

Reply:
before we were checking to ensure that `operationDoc` wasn't falsey.